src/earn/EarnActivePool.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import SkeletonPlaceholder from 'src/components/SkeletonPlaceholder'
+import TokenDisplay from 'src/components/TokenDisplay'
+import { useAavePoolInfo } from 'src/earn/hooks'
+import UpwardGraph from 'src/icons/UpwardGraph'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { useTokenInfo } from 'src/tokens/hooks'
+import networkConfig from 'src/web3/networkConfig'
+import { Address } from 'viem'
+
+function PoolDetailsLoading() {
+  return (
+    <View>
+      <SkeletonPlaceholder backgroundColor={Colors.gray2} highlightColor={Colors.white}>
+        <View
+          style={{
+            ...typeScale.labelXSmall,
+            width: 64,
+            borderRadius: 100,
+          }}
+        />
+      </SkeletonPlaceholder>
+    </View>
+  )
+}
+
+export default function EarnActivePool() {
+  const { t } = useTranslation()
+  const token = useTokenInfo(networkConfig.arbUsdcTokenId)
+  const asyncPoolInfo = useAavePoolInfo(token?.address as Address)
```

Comment:
`null` is not an Address.  You could, for example, use `isAddress` https://viem.sh/docs/utilities/isAddress.html which implements a type guard and narrows the type to an `Address`.

```ts
const tokenAddress = token.address
if (!tokenAddress || !isAddress(tokenAddress)) {
  return null
}
// ts knows tokenAddress is an Address now
```

Reply:
Fixed in fa05f358887ebb760308ff5fcba477b12c09bbd0!